common-java-libs/org.wso2.apk.apimgt.impl/src/main/java/org/wso2/apk/apimgt/impl/dao/impl/ApplicationDAOImpl.java

```diff
@@ -140,6 +141,19 @@ private String getGroupId(Connection connection, int applicationId) throws SQLEx
         }
         return String.join(",", grpIdList);
     }
+    public Integer getSubscriptionCountByApplicationId(Application application, String organization)
+            throws APIManagementException {
+
+        int subscriptionCount = 0;
+
+        try (Connection connection = APIMgtDBUtil.getConnection()) {
+            return getSubscriptionCountByApplicationId(connection, application, organization);
+        } catch (SQLException e) {
+            handleException("Failed to get SubscribedAPI of : application " + application.getName() + " in " +
+                    "organization " + organization, e);
+        }
+        return subscriptionCount;
```

Comment:
Do we need this? its always 0

Reply:
this method calls for private method and returns the correct count, else will return 0